test/source/symlink-test/symlinked-file-outside-source

```diff
@@ -1 +1 @@
-/etc/passwd
\ No newline at end of file
+/etc/passwd
```

Comment:
Unfortunately I am unable to edit this myself. Please revert this change. The command I found online to help with this is:

```text
git restore -s origin/master test/source/symlink-test/symlinked-file-outside-source
```

This command requires Git 2.23 which [was released in 2019](https://github.com/git/git/blob/master/Documentation/RelNotes/2.23.0.txt), so you should have this command.

Reply:
@ashmaroli The original change prepended `C:` here. If there's a way to make this agnostic for Windows (like creating the symlink when the tests are run instead of storing the symlinked file), then we can explore something like that.

Reply:
I reverted the change as requested. I wasn't able to restore the file from my Windows box, though, I had to reboot to my Linux box. Would be great if there were a way to make the test OS agnostic, currently this is a pitfall if using Windows for development.